### PR TITLE
BETA: Register rootcircle.is-a.dev

### PR DESCRIPTION
**Pull request by rootCircle**
Added `rootcircle.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/rootcircle.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "rootCircle",
+        "email": "dev.frolics@gmail.com"
+    },
+    "record": {
+        "CNAME": "rootcircle.github.io"
```
Comment (rootCircle) [inline domains/rootcircle.json:7]: Also, I am not sure atm, that if I should use `rootcircle.github.io` or `rootcircle.github.io/blog` as CNAME. The former redirects to `rootcircle.github.io/blog`.

Can you please help me with that?

Comment (21Z) [reply domains/rootcircle.json:7]: You can't have `/` in `CNAME` so when people visits rootcircle.is-a.dev they'll be redirected to rootcircle.is-a.dev/blog

Comment (rootCircle) [reply domains/rootcircle.json:7]: okay cool! thanks, this too will work :-)

**Review by iostpa — APPROVED** [review]
seems good to me

**Pull request merged**